packages/smooth_app/lib/database/dao_osm_location.dart

```diff
@@ -69,7 +69,7 @@ class DaoOsmLocation extends AbstractSqlDao {
       final String column,
     ) =>
         e.toString().startsWith(
-            'DatabaseException(duplicate column name: $column (code 1 SQLITE_ERROR)');
+            'DatabaseException(duplicate column name: $column (code 1 SQLITE_ERROR[1])');
```

Comment:
```suggestion
            'DatabaseException(duplicate column name: $column (code 1 SQLITE_ERROR');
```

Reply:
Thank you @M123-dev!
Actually it's not a typo, it's an alternate wording: I tested "my" version and the error message matched. On you device it's slightly different.
My suggestion works on both cases.